api/src/main/java/net/md_5/bungee/api/boss/BossBarDivision.java

```diff
@@ -0,0 +1,13 @@
+package net.md_5.bungee.api.boss;
+
+/**
+ * Represents a division (style) of a {@link BossBar}
+ */
+public enum BossBarDivision
```

Comment:
Can remove Boss - its in the package.

Reply:
Also change Dvision to Style here or not?